src/components/Dashboard/Dashboard.js

```diff
@@ -10,13 +11,64 @@ const Dashboard = () => {
   if (!user) {
     navigate('/login')
   }
+  const menu = [
+    {
+      categoryName: "tatli",
+      items: [
+        {
+          itemName: "su",
```

Comment:
Arrayin ismi items olduğu için burada itemName ve itemPrice dememize gerek yok. Direkt name ve price diyebiliriz.